### PR TITLE
Updated To Newer Version Of Sentry SDK And Laravel

### PR DESCRIPTION
**Pull request by sentryadam0000345**
sentry/sentry-laravel: 1.8

laravel/framework: 7.24

php: 7.2.5

**Links:**

Unhandled:  https://sentry.io/organizations/testorg-az/issues/1869476088/?query=is%3Aunresolved

Handled:  https://sentry.io/organizations/testorg-az/issues/1869475927/?_allp=1&query=is%3Aunresolved

Checkout:  https://sentry.io/organizations/testorg-az/issues/1869776421/?project=5412551&query=is%3Aunresolved+transaction_id%3A_kg63f32h3

### CODE REVIEW

**Comment by thinkocapo** [comment]
@sentryadam0000345 can you post a link from where you use a React app to hit /checkout?

The [Checkout link](https://sentry.io/organizations/testorg-az/issues/?query=is%3Aunresolved+transaction_id%3A_igm6dk7gz) you shared, the error looks great but when I click transaction_id I should be able to find the corresponding React error. The URL tag on that react error needs to be your React app as served by Cloud Run (the URL will show that). 

In the link you provided, it looks like you are making the request from the React app served by Cloud Run? Can you confirm?
image

if this is the case, then there's something that needs investigation here, to get the transaction_id shared between both errors. Let me know if you need a hand, after you try debugging this. Let me know if any of this does not make sense.

**Comment by sentryadam0000345** [comment]
@thinkocapo It is working, here is a link to the React app:

https://adamtoth-fejel-react-errors-laravel-edition-m3uuizd7iq-uc.a.run.app/

Please use this link for Checkout instead:

https://sentry.io/organizations/testorg-az/issues/1869776421/?project=5412551&query=is%3Aunresolved+transaction_id%3A_kg63f32h3

I have updated the above with this link as well.

File: config/cache.php [hunk 1]
```diff
@@ -70,7 +74,16 @@
 
         'redis' => [
             'driver' => 'redis',
-            'connection' => 'default',
+            'connection' => 'cache',
+        ],
+
+        'dynamodb' => [
+            'driver' => 'dynamodb',
+            'key' => env('AWS_ACCESS_KEY_ID'),
```
Comment (thinkocapo) [inline config/cache.php:82]: can we remove all this about aws secret keys?

File: app/Providers/RouteServiceProvider.php [hunk 2]
```diff
@@ -16,6 +16,13 @@ class RouteServiceProvider extends ServiceProvider
      */
     protected $namespace = 'App\Http\Controllers';
 
+    /**
+     * The path to the "home" route for your application.
+     *
+     * @var string
+     */
+    public const HOME = '/home';
```
Comment (thinkocapo) [inline app/Providers/RouteServiceProvider.php:24]: assuming this does not mean /home on my unix machine. What is this relative to? Is this significant? Do I have to clone this app into '/home' or anyting like that?

File: config/mail.php [hunk 3]
```diff
@@ -4,45 +4,73 @@
...
     |
     */
 
-    'port' => env('MAIL_PORT', 587),
+    'mailers' => [
+        'smtp' => [
+            'transport' => 'smtp',
+            'host' => env('MAIL_HOST', 'smtp.mailgun.org'),
+            'port' => env('MAIL_PORT', 587),
+            'encryption' => env('MAIL_ENCRYPTION', 'tls'),
```
Comment (thinkocapo) [inline config/mail.php:41]: I believe this is all boilerplate stuff that came with your Laravel upgrade. What do you think @sentryadam0000345 

I'd rather just remove this if possible, as now bots have added us to their list of repo's to scan for sensitive information.

File: routes/web.php [hunk 4]
```diff
@@ -1,47 +1,63 @@
...
     } catch (\Throwable $exception) {
         report($exception);
     }
     return $exception;
-});
+}]);
 
-Route::get('/unhandled', function () {
+Route::get('/unhandled', ['as' => 'unhandled', function () {
+    //
```
Comment (thinkocapo) [inline routes/web.php:34]: can remove this extra `//`

**Comment by sentryadam0000345** [comment]
@thinkocapo Did your recommended changes, let me know if there is anything else I should take care of.

**Review by thinkocapo — APPROVED** [review]
merge it

**Pull request merged**